src/Components/App/App.js

```diff
@@ -24,15 +25,18 @@ const App = () => {
     setActivity(activity);
   }
 
-  useEffect(() => {
-    filterActivities();
-  }, [feeling]);
-
   const filterActivities = () => {
-    const filtered = activities.filter(act => act.feelings.includes(feeling));
+    const feelingCategory = feelingsData
+      .find(({ associatedFeelings }) => associatedFeelings.includes(feeling))
+      ?.feeling.toLowerCase();
```

Comment:
The optional chaining makes a lot more intuitive sense to me here than in that h2 in activities. Pretty cool!

Reply:
++++++++